### DISCUSSION TITLE
How to capture data inside a Collection View?

### DISCUSSION THREAD

**Discussion by Colima-97** [dimonovdd/ImageFromXamarinUI#6] [Q&A]

**Comment by dimonovdd** [comment]
Collection View has caching strategies. And all items are not rendered.

This plugin finds the native control that was created to render the Xamarin.Forms control and just captures it using native APIs.

 I don't think it will be possible to capture the all  list correctly.

**Reply by Colima-97** [reply]
You won't believe me but, **_I did it work on iOS_**, however, I have a bug and the last item is not showing completely.

I'll try to fix it and hopefully, I'll share it here soon 😎

**Reply by dimonovdd** [reply]
@Colima-97 It will be cool if you do it. I'm a little busy on another plugin. But I will consider your PR
